README.md

```diff
@@ -36,17 +36,17 @@ contained here, so take a moment to browse through the various files and see
 what's available:
 ```
```

Comment:
> We start by first cloning this repository and `cd`ing into its `demo` subdirectory.

We don't cd into the `demo` subdirectory anymore.

Reply:
Ack

Reply:
this is fixed in the latest push